defaults/main.yml

```
@@ -3,8 +3,12 @@ kube_config_dir: "/etc/kubernetes"
 kube_cert_dir: "/etc/kubernetes/ssl"
 kube_manifests_dir: "/etc/kubernetes/manifests"
 kubernetes_service_addresses: "172.21.0.0/16"
-kubernetes_version: "1.4.1"
-hyperkube: "gcr.io/google_containers/hyperkube:v{{kubernetes_version}}"
+
```

Using kubelet variables is misleading. I would simply change this to --
```suggestion
kubernetes_version: "1.14.3"
hyperkube_repo: "gcr.io/google_containers"
hyperkube: "{{hyperkube_repo}}/hyperkube:v{{kubernetes_version}}"
```

---

Those are naming conventions we use across other repos, we should keep it consistent.
For gcr_repo, we use it for other components along with hyperkube/kubelet.

---

One second thought made the change since hyperkube though would be at same version as kubelet, it's better to distinguish.